app/views/hackathons/submissions/new.html.erb

```diff
@@ -1,25 +1,25 @@
-<% content_for :title, "Apply to list your hackathon" %>
+<% content_for :title, "List your hackathon on hackathons.hackclub.com" %>
 <% image = "https://cloud-rmjrb2y89-hack-club-bot.vercel.app/0oy1pj1mznloa96gh1sg-oiontn0tkxelxwjutwobkza.jpeg" %>
 
 <%= render "shared/full_page_card", background_url: image do %>
   <%= image_tag "flag-standalone.svg", style: "max-width: 30%; margin-bottom: var(--spacing-3)" %>
   <h1>
-    Apply to list your hackathon
+    List your hackathon on <%= link_to "hackathons.hackclub.com", Hackathons::WEBSITE, target: "_blank" %>.
   </h1>
 
-  <div class="mb4">
+  <div class="mb2">
     <p>
       If you'd like to list your hackathon on
       <%= link_to "hackathons.hackclub.com", Hackathons::WEBSITE, target: "_blank" %>,
       please fill out this form! We will get back to you within a couple days.
     </p>
 
-    <p class="mb0">
```

Comment:
i thought that this made the whole thing more readable and more consistent

Reply:
If we need some general spacing on the bottom, we should probably have that on the general container/layout.